### PR TITLE
BaSP-TG-68: Get/Delete Projects

### PR DESCRIPTION
**Pull request by maxidelfino**

### CODE REVIEW

File: src/Components/Shared/Table/index.js [hunk 1]
```diff
@@ -1,6 +1,6 @@
 import React from 'react';
 import styles from './table.module.css';
-import Delete from '../../../assets/trash.png';
+import Delete from '../../../Assets/close_cross.png';
```
Comment (lmalvasia) [inline src/Components/Shared/Table/index.js:3]: The folder should be `assets` (lower case).
Why did you change the delete icon?

File: package.json [hunk 2]
```diff
@@ -16,7 +16,7 @@
     "web-vitals": "^1.1.2"
   },
   "scripts": {
-    "start": "react-scripts start",
+    "start": "react-scripts --openssl-legacy-provider start",
```
Comment (lmalvasia) [inline package.json:19]: Remove this

**Review by lmalvasia — APPROVED** [review]

File: src/redux/projects/actions.js [hunk 3]
```diff
@@ -0,0 +1,104 @@
+import {
+  GET_PROJECTS_PENDING,
+  GET_PROJECTS_SUCCESS,
+  GET_PROJECTS_ERROR,
+  DELETE_PROJECTS_PENDING,
```
Comment (baudraccoverona) [inline src/redux/projects/actions.js:5]: Please remove the plural from all the delete, post and put

File: src/redux/projects/constants.js [hunk 4]
```diff
@@ -0,0 +1,30 @@
+const GET_PROJECTS_PENDING = 'GET_PROJECTS_PENDING';
+const GET_PROJECTS_SUCCESS = 'GET_PROJECTS_SUCCESS';
+const GET_PROJECTS_ERROR = 'GET_PROJECTS_ERROR';
+
+const DELETE_PROJECTS_PENDING = 'DELETE_PROJECTS_PENDING';
```
Comment (baudraccoverona) [inline src/redux/projects/constants.js:5]: Same here

**Review by baudraccoverona — APPROVED** [review]
